flatfs.go

```diff
@@ -35,6 +35,9 @@ var (
 	// DiskUsageFile is the name of the file to cache the size of the
 	// datastore in disk
 	DiskUsageFile = "diskUsage.cache"
+	// DiskUsageNodes stores notes about the accuracy of the size of
+	// the diskusage calculation
+	DiskUsageNotes = "diskUsage.notes"
```

Comment:
Any reason not to put this in the same file? Or just have one JSON/CBOR file with all the information (significantly more flexible)? That would also allow it to be atomic.

This would also allow us to keep track of the disk size accuracy if we crash. We could record the fact that we're currently running *while* running. If we start up and see that we didn't correct the state, we can adjust the accuracy to approximate (or something else).

Reply:
I did it this way for simplicity.  It has the advantage of not having to read the accuracy data once written, but I have no problem doing something more sophisticated.

Reply:
My motivation is really atomicity. It allows us to atomically store the repo size *and* it's accuracy (and allows us to deal with the inaccurate repo size on crash case). However, I don't really feel strongly any which way.